docs/chatterino.d.ts

```diff
@@ -12,13 +12,43 @@ declare module c2 {
     channel_name: String;
   }
 
+  enum Platform {
+    Twitch,
+    IRC,
+  }
+  enum ChannelType {
+    None,
+    Direct,
+    Twitch,
+    TwitchWhispers,
+    TwitchWatching,
+    TwitchMentions,
+    TwitchLive,
+    TwitchAutomod,
+    Irc,
+    Misc,
+  }
+
+  interface IWeakResource {
+    is_valid(): boolean;
+  }
+
+  class Channel implements IWeakResource {
+    is_valid(): boolean;
+    get_name(): string;
+    get_type(): ChannelType;
+    get_display_name(): string;
+    is_twitch_channel(): boolean;
+
+    static by_name(name: string, platform: Platform): null | Channel;
+    static by_twitch_id(id: string): null | Channel;
+  }
```

Comment:
Isn't there `send_message` and `add_system_message` missing, because `send_msg` and `system_msg` were removed a few lines below this class definition and in the `plugin-meta.lua` there are those functions defined as member of `Channel`?

Reply:
They are missing, thank you for reminding me of that